Xamarin.Forms.Core/Label.cs

```diff
@@ -210,7 +218,7 @@ public int MaxLines
 			InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
 
 		void IFontElement.OnFontChanged(Font oldValue, Font newValue) =>
-         	InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
+			 InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
```

Comment:
meh

Reply:
😱 done!

Reply:
Tabs vs Spaces, I spent so much time fixing those so it doesn't get committed, but I missed one